Makefile

```diff
@@ -26,12 +26,12 @@ setup_nextest:
 	@which cargo-nextest >/dev/null || cargo install cargo-nextest
 
 test: setup_nextest ## Run tests single threaded due to s3 race conditions
-	AWS_REGION=us-east-2 \
```

Comment:
This part allows to perform tests on real aws environment. i think we should keep it.